### PR TITLE
Add FFI for tokenize_binding()

### PR DESCRIPTION
**Pull request by dennisschagt**
Small continuation of work done in https://github.com/newsboat/newsboat/pull/2586.

### CODE REVIEW

File: src/keymap.cpp
Comment (dennisschagt) [inline src/keymap.cpp:736]: This has no user-observable effect yet until we register the keymap as a handler for "bind" in `Controller` and `PbController`:
https://github.com/newsboat/newsboat/blob/ca977d51a0ea9bc08d302d9ce58e424090823a87/src/controller.cpp#L168-L172

https://github.com/newsboat/newsboat/blob/ca977d51a0ea9bc08d302d9ce58e424090823a87/src/pbcontroller.cpp#L260-L261

File: rust/libnewsboat-ffi/src/keymap.rs
Comment (Minoru) [inline rust/libnewsboat-ffi/src/keymap.rs:79]: Nitpick: It's better to do `unwrap_or_else(|| String::new())`, which avoids allocating a new `String` unless it's really needed. I'm surprised Clippy didn't suggest it in CI.

**Review by Minoru — APPROVED** [review]
Spoon-feeding me the changes works much better than waiting for a review on a large PR. I'm glad you've come up with this :)

Thanks for the work! I only have one nitpick, which you're free to ignore. Please merge this now or after addressing the nitpick.

File: rust/libnewsboat-ffi/src/keymap.rs
Comment (dennisschagt) [reply rust/libnewsboat-ffi/src/keymap.rs:79]: It looks like `String::new()` does not allocate any buffer
From https://doc.rust-lang.org/std/string/struct.String.html#method.new:
> Given that the String is empty, this will not allocate any initial buffer.

[A clippy lint](https://github.com/rust-lang/rust-clippy/issues/2972) was added a while back to suggest `String::new()` over `"".to_string()`
However, that was later changed to a "pedantic" lint.

I will leave it as-is for now, but thanks for the suggestion! It is nice to be aware of the `_or_else()` variant of unwrapping. I will try to keep it in mind for situations where it makes sense.

**Comment by dennisschagt** [comment]
Thanks!

> Spoon-feeding me the changes works much better than waiting for a review on a large PR.

I'm happy to hear that.
It also feels a lot nicer to me to get things merged instead of slowly working through the checkboxes on the original PR, so I will try to keep these PRs small

**Pull request merged**
